v03_pipeline/lib/tasks/reference_data/updated_cached_reference_dataset_query.py

```diff
@@ -71,6 +72,14 @@ def requires(self) -> luigi.Task:
                     ],
                 ),
             )
+        # Special nested import to avoid a circular dependency issue
+        # (ValidateCallset -> this file -> UpdatedReferenceDatasetCollection -> ValidateCallset)
+        # The specific CRDQ referenced in ValidateCallset will never reach
+        # this line due to it being a "query_raw_dataset"
+        from v03_pipeline.lib.tasks.reference_data.updated_reference_dataset_collection import (
```

Comment:
Calling this out as weird/off.  We have a pseudo-circular dependency in the pipeline, in that `ValidateCallsetTask` depends on an updated validation hail table, which depends on `gnomad` versioning.  pulling the `import` down into the `requires()` fixes the issue and was the cleanest and easiest way forwards.  The _best_ way forward is to split the crdq task into two tasks, one which depends on the `combined` table and one that doesn't.  I left that as out of scope.

Reply:
might be worth leaving a note in the comment saying that this could be resolved by splitting the crdq task?